docs/administer/default_policies.md

```diff
@@ -0,0 +1,88 @@
+# Default configuration for Pod scheduling policies
+
+In Percona Everest, the default pod scheduling policies are **preset** rules that help ensure optimal placement of database components across a Kubernetes cluster. 
+
+## Importance of default Pod scheduling policies
+
+Percona Everest Administrators create the **default Affinity** rules. Thus, Percona Everest users can use these **predefined** settings without the need to create custom rules for every database cluster they set up. Also, these Pod scheduling policies are defined to align with Percona Everest's infrastructure and internal policies. 
+
+## Common attributes of default Pod scheduling policies
+
+The following are the common attributes of a default policy for all three database technologies.
+
+- **Type**: The Affinity Type applied is **Pod Anti-Affinity**. This ensures that pods of the same component are not co-located on the same node.
+
+- **Preference**: **Preferred 1** means the scheduler will try to satisfy this rule but will still schedule the pod even if the condition cannot be met.
+
+- **Topology Key**: The topology key `kubernetes.io/hostname` defines the scope of the rule. In this case, it ensures that the anti-affinity is evaluated at the node level, preventing matching pods from being placed on the same node.
+
+
+These policies use pod labels to identify which pods should not be co-located. The scheduler will try to honor the rule but will not enforce the separation of pods.
+ 
+##  Default policy components by database technology
+
+The **Components** for a default policy change as per the technology:
+
+### Components for MYSQL database
+
+- **DB Node**: The core component that stores and serves data.
+
+- **Proxy**: Acts as a load balancer and router for DB Nodes.
+
+Default anti-affinity prevents multiple DB Nodes or Proxies of the same cluster from being scheduled on the same node.
```

Comment:
`Default anti-affinity` - it outlines that exactly "default policy" makes anti-affinity. But Custom policy can be configured to achieve the same result.

Reply:
Yes. But Peter wanted me to highlight the Default policies, so I added. We can write a note saying that its applicable to custompolicies as well.

Reply:
@maxkondr if the same default is already there. Why would a user create the same custom policy?